tests/upgrades/test_capsule.py

```diff
@@ -178,41 +176,54 @@ def test_post_user_scenario_capsule_sync_yum_repo(self, request, target_sat, def
 
         """
         request.addfinalizer(lambda: cleanup(content_view, repo, product))
-        repo_name = gen_string('alpha')
-        rpm_name = rpm2.split('/')[-1]
         activation_key = (
             settings.upgrade.capsule_ak[settings.upgrade.os]
             or settings.upgrade.custom_capsule_ak[settings.upgrade.os]
         )
-        cap_host = settings.upgrade.capsule_hostname
-        ak = target_sat.api.ActivationKey(organization=default_org.id).search(
+        cap_host = Capsule(
+            hostname=settings.upgrade.capsule_hostname
```

Comment:
Same comment as above. We need to determine the capsule from the satellite as the pair of satellite and capsule are integrated from integration stage in CI !

Reply:
Implemented in `pre_configured_capsule` fixture.